book-details.html

```diff
@@ -11,24 +11,29 @@
 </head>
 
 <body>
-  <div class="book-cover-page">
-    <img class="book-img" src="images/book-cover.png" alt="Book Cover" />
-    <div class="book-info">
-      <header class="book-header">
-        <h1 class="book-title">Título del libro</h1>
-        <p class="book-genre">(género)</p>
-      </header>
-      <div class="book-content-row">
-        <h3 class="default-text category">Autor del libro:</h3>
-        <p class="default-text">Nombre del autor del libro</p>
-      </div>
-      <div class="book-content-row">
-        <h3 class="default-text category">Editorial:</h3>
-        <p class="default-text">Nombre de la editorial</p>
-      </div>
-      <div class="book-content-row">
-        <h3 class="default-text category">Año de publicación:</h3>
-        <p class="default-text">Año de publicación</p>
+  <div class="book-detail-container">
+    <a href="./index.html">
+      <p class="back-button">Atrás</p>
+    </a>
+    <div class="book-cover-page">
+      <img class="book-img" src="images/book-cover.png" alt="Book Cover">
```

Comment:
```suggestion
      <img class="book-img" src="images/book-cover.png" alt="Book Cover" />
```

Reply:
Check scroll on details page, you should not have it

Reply:
The scroll was made by the fixed height that I used. When I remove that, the scroll just go away. Thank you!